qiskit/qobj/converters/pulse_instruction.py

```diff
@@ -558,6 +560,7 @@ def gen_sf_schedule(*args, **kwargs):
                 _frequency = frequency_expr(*args, **kwargs)
                 return instructions.SetFrequency(_frequency, channel) << t0
 
+            # ToDo(4872) remove the use of deprecated constructor Schedule(*schedulers)
```

Comment:
I think there was a bug here:
https://github.com/Qiskit/qiskit-terra/pull/5093/files#diff-78f2c68464fc91f622184a3afeaf0a8bR554

https://github.com/Qiskit/qiskit-terra/pull/5093/files#diff-78f2c68464fc91f622184a3afeaf0a8bR580

While adding tests I noticed that in case frequency was passed as a string, it still tries to multiply it by 1e9.
I'll add a fix for these as well.

Reply:
Thank you!